app/genesis.go

```diff
@@ -8,6 +8,7 @@ import (
 	cmn "github.com/tendermint/tmlibs/common"
 )
 
+// LoadGenesis - Load the genesis json file
```

Comment:
sure - I guess sometimes the comment doesn't sounds right as an action like the example you've provided, and I tried to keep it consistent this way - but will change here